flight/src/types.rs

```diff
@@ -82,11 +89,34 @@ impl TryFrom<unvalidated::Event<Ipld>> for ConclusionInit {
     type Error = anyhow::Error;
 
     fn try_from(event: unvalidated::Event<Ipld>) -> Result<Self> {
+        // Extract the init payload from the event
         let init_payload = event
             .init_payload()
-            .ok_or_else(|| anyhow!("no init payload found"))?;
+            .ok_or_else(|| anyhow!("malformed event: no init payload found"))?;
+
+        // Get the model from the init header
+        // The model indicates the creator of the stream
+        let model = init_payload.header().model();
+
+        // Define the metamodel stream ID
+        let meta_model_stream_id = StreamId::from_str(METAMODEL_STREAM_ID)?;
```

Comment:
can we make *this* the constant we define at the top of the file?  Instead of converting from string to StreamId for every event.

Reply:
I did this here, https://github.com/ceramicnetwork/rust-ceramic/pull/529 turns out it was a bit more complicated that I expected but we have it now.